jsapp/js/app.es6

```diff
@@ -1599,7 +1605,8 @@ var routes = (
         <DefaultRoute handler={FormLanding} />
       </Route>
 
-      <DefaultRoute handler={FormsSearchableList} />
+      {/* <DefaultRoute handler={FormsSearchableList} /> */}
```

Comment:
I commented this out because I don't know how important it is or if we'll need it in the future